### ISSUE TITLE
As a system admin, I want to select what fields require additional approval if a correction is required 

### ISSUE THREAD

**Issue by jpye-finch** [opencrvs/opencrvs-core#2726]
### Description

- Adds Toggle to RightSideMenu. Default state off. 
- OnToggle to On. Then the field requires additional approval if a correction is required/requested. (Correct Record)

### Acceptance criteria

GIVEN
WHEN
THEN

### Design
https://www.figma.com/file/pyp3KyEVRMjkmK2ye2iT9W/Configuration?node-id=1099%3A60456

### Dev-tasks

- [ ]  Task 1

**Comment by jpye-finch** [comment]
Yes, this came up in convo with Ed. Not a priority until a country asks for it

**Comment by jpye-finch** [comment]
have added requirement to Correct Record v2 https://app.zenhub.com/workspaces/opencrvs-core-5afeded64a54b06b54a969b2/issues/opencrvs/opencrvs-core/3601

closing

**Comment by anny320** [comment]
closed
